packages/components/src/components/AlertDialog/AlertDialog.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { Default } from "./AlertDialog.stories";
+
+const OPEN_ALERT_DIALOG_TEXT = "Open alert dialog";
+
+describe("<AlertDialog />", () => {
+  it("renders correctly", async () => {
+    render(<Default />);
+    const renderedOpenButton = screen.getByText(OPEN_ALERT_DIALOG_TEXT);
+    expect(renderedOpenButton).toBeInTheDocument();
+
+    await userEvent.click(renderedOpenButton);
+    expect(screen.getByRole("dialog")).toBeInTheDocument();
```

Comment:
One more thing. Sorry. Can you test to make sure the `esc` key doesn't close the AlertDialog?